server/spinwick.go

```diff
@@ -32,13 +40,27 @@ func (s *Server) handleCreateSpinWick(pr *model.PullRequest, size string, withLi
 		return
 	}
 
-	if withLicense {
-		s.sendGitHubComment(pr.RepoOwner, pr.RepoName, pr.Number, "Creating a new HA SpinWick test server using Mattermost Cloud.")
-	} else {
-		s.sendGitHubComment(pr.RepoOwner, pr.RepoName, pr.Number, "Creating a new SpinWick test server using Mattermost Cloud.")
+	request := &spinwick.Request{
+		InstallationID: "n/a",
+		Error:          nil,
+		ReportError:    false,
+		Aborted:        false,
+	}
+
+	// for Mattermost webapp and mattermost server, business as usual
+	if pr.RepoName == "mattermost-webapp" || pr.RepoName == "mattermost-server" {
```

Comment:
we can have other repos that spin MM servers not only "mattermost-webapp" or "mattermost-server"